src/systems/filecoin_vm/runtime/impl/runtime.go

```diff
@@ -121,6 +121,8 @@ type VMContext struct {
 	_toplevelBlockWinner addr.Address
 	// Top-level call sequence number of the "From" actor in the initial on-chain message.
 	_toplevelSenderCallSeqNum actstate.CallSeqNum
```

Comment:
Can you remove this? Or perhaps rename and fix caller to use the message call seq num, instead of passing the whole message here.